src/test/kotlin/org/rust/ide/wordSelection/RsBlocksAndBodiesSelectionHandlerTest.kt

```diff
@@ -0,0 +1,368 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.wordSelection
+
+class RsBlocksAndBodiesSelectionHandlerTest : RsSelectionHandlerTestBase() {
```

Comment:
I don't sure we should create a separate test file for it. I mean, maybe it's better to make common SelectionHandlerTest and write all tests there. Anyway, this name is not true, because you also tests grouping =)

Reply:
One long test case is quite hard to read and to orient inside. It's quite helpful to me to group various testing aspects into separate test cases, this one is generic tests for blocks and bodies. 
I'd even split methods and structs, but decided not to, until we have more tests